Content.Shared/Gravity/SharedGravitySystem.cs

```diff
@@ -31,8 +31,9 @@ public bool IsWeightless(EntityUid uid, PhysicsComponent? body = null, Transform
                 return true;
 
             // If grid / map has gravity
-            if (TryComp<GravityComponent>(xform.GridUid, out var gravity) && gravity.Enabled ||
-                 TryComp<GravityComponent>(xform.MapUid, out var mapGravity) && mapGravity.Enabled)
+            if (xform.MapID == MapId.Nullspace ||
```

Comment:
This seems kinda suss when floating visualizer shouldn't be applying in nullspace?